src/pages/guidelines/spacing.mdx

```diff
@@ -0,0 +1,126 @@
+```
+<br />
+
+Below is the example of a regular image card on the left and how the spacing tokens are applied to it on the right.
+
+<Row>
+<Column colLg={8}>
+
+![Example of spacing tokens applied](../../images/guidelines/spacing/spacing-example-01.jpg)
+
+
+
+</Column>
+</Row>
+
+<br />
+
+### Spacing scale
+
+Use the spacing scale when building individual components. It includes small
+increments needed to create appropriate spatial relationships for detail-level
+designs. This scale is applied and used within all Carbon components.
+
+<div className="spacing-table">
+
+| Token         | rem   | px  | Example                                       |
+| ------------- | ----- | --- | --------------------------------------------- |
+| `$spacing-01` | 0.125 | 2   | ![spacing-3xs example](../../images/guidelines/spacing/spacing-01.svg) |
+| `$spacing-02` | 0.25  | 4   | ![spacing-2xs example](../../images/guidelines/spacing/spacing-02.svg) |
+| `$spacing-03` | 0.5   | 8   | ![spacing-xs example](../../images/guidelines/spacing/spacing-03.svg) |
+| `$spacing-04` | 0.75  | 12  | ![spacing-sm example](../../images/guidelines/spacing/spacing-04.svg) |
+| `$spacing-05` | 1     | 16  | ![spacing-md example](../../images/guidelines/spacing/spacing-05.svg) |
+| `$spacing-06` | 1.5   | 24  | ![spacing-lg example](../../images/guidelines/spacing/spacing-06.svg) |
+| `$spacing-07` | 2     | 32  | ![spacing-xl example](../../images/guidelines/spacing/spacing-07.svg) |
+| `$spacing-08` | 2.5   | 40  | ![spacing-2xl example](../../images/guidelines/spacing/spacing-08.svg) |
+| `$spacing-09` | 3     | 48  | ![spacing-3xl example](../../images/guidelines/spacing/spacing-09.svg) |
+| `$spacing-10` | 4     | 64  | ![spacing-4xl example](../../images/guidelines/spacing/spacing-10.svg) |
+| `$spacing-11` | 5     | 80  | ![spacing-5xl example](../../images/guidelines/spacing/spacing-11.svg) ||
+| `$spacing-12` | 6     | 96  | ![spacing-6xl example](../../images/guidelines/spacing/spacing-12.svg) |
+| `$spacing-13` | 10    | 160 | ![spacing-7xl example](../../images/guidelines/spacing/spacing-13.svg) |
+
+</div>
+
+<br />
+
+## FAQ
+
+#### Are there any differences when designing with expressive type sets vs. productive type sets?
+
+Yes, please see the [Type pairings and spacing guideline](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/guidelines/type-pairing-and-spacing) to learn more.
+
+#### Are there any other spacing options besides the one listed in the Spacing scale and token section?
+
+Yes, a few other non-token methods can be used for spacing elements, and they each have their unique use case.
+
+* **Center** is used to center an element between two edges fluidly.
+* **Auto** is used on one side of an element to allow undefined space to grow and shrink according to screen size automatically. Typically used for asymmetrical fluid spacing.
+* **Gutter** is used to space items between the grid's 12 columns.
```

Comment:
@photodow These options are listed on the Carbon spacing page as well. I thought it might be helpful, but I get what you are saying. I wonder if the information triggers confusion? Maybe we take it off? Thoguhts?